### PR TITLE
Fix model serialization in MCFL python notebook (fixes #5460)

### LINKED ISSUE #5460: ValueError: bytes is not a 16-char string
## Description

The return statement `return deserialize(pb)` in function `get_model(grid_address, worker_id, request_key, model_id)` utilise in the Federated Learning model-centric example `PySyft/examples/federated-learning/model-centric/mcfl_create_plan.ipynb` gives an error `ValueError: bytes is not a 16-char string`

## Software Versions
Pygrid (latest dev branch)
Syft (0.5.0rc1)
Torch (1.8.0)
Torchvision (0.8.2)
Python - (3.7.6)

## How to Reproduce
1. Start Pygrid domain server using command `./run.sh --host localhost  --port 7000 --start_local_db`
2. Execute the NoteBook `PySyft/examples/federated-learning/model-centric/mcfl_create_plan.ipynb`
3. The following error occurs
```
 # Model 
model_params_downloaded = get_model(grid_address, worker_id, request_key, model_id) 
print("Params shapes:", [p.shape for p in model_params_downloaded])                                                                                            
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-22-14a183e0aa75> in <module>
      1 # Model
----> 2 model_params_downloaded = get_model(grid_address, worker_id, request_key, model_id)
      3 print("Params shapes:", [p.shape for p in model_params_downloaded])

<ipython-input-21-43c3f622395f> in get_model(grid_address, worker_id, request_key, model_id)
      6     pb = ListPB()
      7     pb.ParseFromString(req.content)
----> 8     return deserialize(pb)
      9 

~/anaconda3/lib/python3.7/site-packages/syft/core/common/serde/deserialize.py in _deserialize(blob, from_proto, from_bytes)
     87         traceback_and_raise(deserialization_error)
     88 
---> 89     res = _proto2object(proto=blob)
     90     return res

~/anaconda3/lib/python3.7/site-packages/syft/lib/python/list.py in _proto2object(proto)
    156     @staticmethod
    157     def _proto2object(proto: List_PB) -> "List":
--> 158         id_: UID = deserialize(blob=proto.id)
    159         value = []
    160         # list comprehension doesn't work since it results in a

~/anaconda3/lib/python3.7/site-packages/syft/core/common/serde/deserialize.py in _deserialize(blob, from_proto, from_bytes)
     87         traceback_and_raise(deserialization_error)
     88 
---> 89     res = _proto2object(proto=blob)
     90     return res

~/anaconda3/lib/python3.7/site-packages/syft/core/common/uid.py in _proto2object(proto)
    189             if you wish to deserialize an object.
    190         """
--> 191         return UID(value=uuid.UUID(bytes=proto.value))
    192 
    193     @staticmethod

~/anaconda3/lib/python3.7/uuid.py in __init__(self, hex, bytes, bytes_le, fields, int, version, is_safe)
    167         if bytes is not None:
    168             if len(bytes) != 16:
--> 169                 raise ValueError('bytes is not a 16-char string')
    170             assert isinstance(bytes, bytes_), repr(bytes)
    171             int = int_.from_bytes(bytes, byteorder='big')

ValueError: bytes is not a 16-char string
```

## Expected Behavior
Show the expected model information.

## System Information
 - OS: [ubuntu]
 - OS Version: [18.04]
 - Language Version: [Python 3.7.6]

### PR DESCRIPTION
**Pull request by vvmnnnkv**
## Description
Model serialization scheme was updated in the last moment before 0.5.0rc1, 
but I forgot to change it in part of the MCFL notebook that demonstrates low-level interaction with Grid where model ser-de was made "manually" not using syft routines.
This change doesn't affect any functionality.

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
Ran create plan & execute plan notebooks against pygrid 0.5.0rc1

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

**Review by tudorcebere — APPROVED** [review]

**Review by tudorcebere — APPROVED** [review]

**Comment by madhavajay** [comment]
@vvmnnnkv Can we get these changes added to the `tests/syft/core/fl/model-centric/mcfl_create_execute_plan_test.py` since its a combination of both notebooks?

File: examples/federated-learning/model-centric/mcfl_create_plan.ipynb [hunk 1]
```diff
@@ -48,21 +46,22 @@
     "from syft.lib.python.int import Int\n",
     "from syft.lib.python.list import List\n",
     "from syft.proto.core.plan.plan_pb2 import Plan as PlanPB\n",
-    "from syft.proto.lib.python.list_pb2 import List as ListPB"
+    "from syft.federated.model_serialization import deserialize_model_params\n",
```
Comment (madhavajay) [inline examples/federated-learning/model-centric/mcfl_create_plan.ipynb:49]: Add these changes to `tests/syft/core/fl/model-centric/mcfl_create_execute_plan_test.py`?

**Review by madhavajay — CHANGES_REQUESTED** [review]
Looks good just need to make sure it passes in the test suite as well.

**Review by koenvanderveen — APPROVED** [review]

**Comment by madhavajay** [comment]
Closing due to inactivity.

**Pull request closed without merging**